matching-core/src/main/java/com/graphhopper/matching/LocationIndexMatch.java

```diff
@@ -52,16 +52,23 @@ public LocationIndexMatch(GraphHopperStorage graph, LocationIndexTree index) {
         this.index = index;
     }
 
-    public List<QueryResult> findNClosest(final double queryLat, final double queryLon, final EdgeFilter edgeFilter,
-                                          double gpxAccuracyInMetern) {
+    /**
+     * Returns all edges whose starting points are within the specified radius around the
```

Comment:
There is no guarantee if the distance is large than the cell radius which is usually not the case